impress-org/givewp
# Feature: Add summer sales banner to plugin page

## Description
This PR adds a summer sales banner to the plugin page. There are two different banners designs which can be viewed [here](https://www.figma.com/file/QJRUKnl10E8eFsvSaW5cI4/Stellar%3A-Sale-'23?node-id=1482%3A429&mode=dev). 

The banner populated is determined by the active licenses the user currently has. The licenses being checked for are the add ons listed below : 
- Recurring donations
- Form field manager
-  Fee recovery
-  Manual donations
-  P2P

### Behavior
- The banners has a start date of 07-24-23 and an expiration date of 07-31-23.
- When closed with the icon at the top right. The banner remains closed.
- In mobile view the secondary message is removed from display and the abstract pattern design is moved downward.


## Additional context
The Give links will not be active until day of display.

## Affects

Plugins Page

## Visuals



## Testing Instructions
Will require active licenses for the add ons previously mentioned to accurately test. 

- Visit the Plugin page and view the banner. 
- Test closing the banner and ensuring it does not repopulate after.
- Test all links on the banner.
- Change screen size of the banner.
- Active add ons required with licensing and verify the additional banner displays when all licensing requirements are met.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205113834227970
